src/panel/context/Devtools.tsx

```diff
@@ -66,22 +76,51 @@ export const DevtoolsProvider: FC = ({ children }) => {
 
   // Listen for client init connection
   useEffect(() => {
-    addMessageHandler((message) => {
+    return addMessageHandler(async (message) => {
```

Comment:
How come this is now async?

Should the [related mocks](https://github.com/FormidableLabs/urql-devtools/blob/69ed737f71f660428134077aa2543232f60ad00c/src/panel/cosmos.decorator.tsx#L35) now return promises for consistency?

Reply:
Sorry this was left over from when I was calling getExchangeVersion in here

```suggestion
    return addMessageHandler((message) => {
```

The argument type would still be `() => void` because `addMessageHandler` doesn't await / use the returned value